src/renderer.js

```diff
@@ -57,7 +57,7 @@ setButton.addEventListener("click", () => {
   // Append the new <li> element to the <ul> element
 
   if (isTomorrow) {
-    appendTaskToTomorrowList(newTaskItem);
+    tomorrowTasksList.appendChild(newTaskItem);
```

Comment:
This fixed the bug, maybe the function couldn't get the data from the input fields or something, I'm not sure.